x-pack/docs/en/security/authentication/user-lookup.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[[user-lookup]]
+=== Looking up users without authentication
+
+{es} <<realms,realms>> exist primarily to support
+<<setting-up-authentication,user authentication>>. 
+Some realms authenticate users via a password (for example the
+<<native-realm,`native`>> and <<ldap-realm,`ldap`>> realms) and other realms use
+more complex authentication protocols (for example the <<saml-realm,`saml`>> and
+<<oidc-realm,`oidc`>> realms).
+In each case, the _primary_ purpose of the realm is to establish the identity of
+the user who has made a request to the {es} API.
+
+However, some {es} features need to _lookup_ a user without using their credentials.
+
+- The <<run-as-privilege,`run_as`>> feature executes requests on behalf of
+  another user. An authenticated user with `run_as` privileges can perform
+  requests on behalf of another unauthenticated user.
+
+- The <<authorization_realms,delegated authorization>> feature links two realms
+  together so that a user who authenticates against one realm can have the roles
+  and metadata associated with a user from a different realm.
+
+In each of the above cases the second user must be resolved without directly
```

Comment:
Suggestion for a bit more clarity..

In each of the above cases a user must first authenticate to one realm, and then authorization is handled by looking up that user in a second realm. The user is authenticated in the first realm, but only must exist in the second realm (i.e. the user does not credentials to the second realm). Querying the second realm for authorization purposes is known as user lookup. 

See the <<run-as-privilege,`run_as`>> and <<authorization_realms,delegated authorization>> documentation for which realms support the ability to perform run_as or delegate authorization. In both cases the only following realms can be for the user lookup.

Reply:
resolving as superseded by Adam's suggestion below.